packages/patternfly-4/react-core/src/components/Tabs/Tabs.md

```diff
@@ -449,22 +449,63 @@ class TabsNavVariant extends React.Component {
 
   render() {
     return (
-      <Tabs
-        activeKey={this.state.activeTabKey}
-        onSelect={this.handleTabClick}
-        aria-label="Local"
-        variant={TabsVariant.nav}
-      >
-        <Tab eventKey={0} title="Tab item 1">
-          Tab 1 section
+       <React.Fragment>
+        <Tabs activeKey={this.state.activeTabKey} onSelect={this.handleTabClick} leftScrollAriaLabel="scrollLeft" rightScrollAriaLabel="right">
+          <Tab eventKey={0} title="Tab item 1">
+            Tab 1 section
         </Tab>
```

Comment:
The `</Tab>` tags are not indented properly.

Reply:
Thanks for the feedback, will work to address your two concerns.